examples/fakeserver/build.gradle

```diff
@@ -27,7 +27,7 @@ sourceCompatibility = '1.8'
 targetCompatibility = '1.8'
 
 dependencies {
-    compile 'com.stellarstation.api:stellarstation-api:0.0.8'
+    compile 'com.stellarstation.api:stellarstation-api:0.3.0'
```

Comment:
Just to follow up on your question on Slack, this file doesn't seem very copy-pastable, so it's probably fine to use `compile project(':api')` here if you wanted.

Reply:
Yeah, updated the file.